services/core-web/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
```

Comment:
common file for generating columns.
Did not refactor _every_ table to use these functions, but some to test it out.

Some fun facts about columns!
The _dataIndex_ determines the first parameter that is passed to the render function. If there is no dataIndex, it will pass the entire record as the first parameter (so you can get rid of unused "text" parameters for things like action columns).
The _key_ is used for things like sorting and drag-dropping, it's good practice to give every column a key.